server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -4,31 +4,97 @@
  * This file generates an index of proptypes by component displayname, slug and folder name
  */
 
+const startTime = process.hrtime();
+
 /**
  * External Dependencies
  */
-
+require( 'babel-register' );
 const fs = require( 'fs' );
 const path = require( 'path' );
 const reactDocgen = require( 'react-docgen' );
+const { getPropertyName, getMemberValuePath, resolveToValue } = require( 'react-docgen/dist/utils' );
+const util = require( 'client/devdocs/docs-example/util' );
 
 const root = path.dirname( path.join( __dirname, '..', '..' ) );
 const pathSwap = new RegExp(path.sep, 'g');
+const handlers = [ ...reactDocgen.defaultHandlers, commentHandler ];
```

Comment:
Nice. But it's still taking quite a long time:

```
nmda:~/a8c/wp-calypso (fix/devdocs-proptype-index=)$ time make server/devdocs/proptypes-index.json
Building: proptypes-index.json
Time: 33s 280.235ms
```

Is this because of the sheer amount of files we're processing? Is there any way we could cut this down?